openpype/hosts/houdini/plugins/create/create_redshift_rop.py

```diff
@@ -76,6 +79,16 @@ def create(self, subset_name, instance_data, pre_create_data):
                     camera = node.path()
             parms.update({
                 "RS_renderCamera": camera or ""})
+
+        if pre_create_data.get("split_render"):
+            rs_filepath = \
+                "{export_dir}{subset_name}/{subset_name}.$F4.rs".format(
+                    export_dir=hou.text.expandString("$HIP/pyblish/rs/"),
+                    subset_name=subset_name,
+                )
+            parms["RS_archive_enable"] = 1
+            parms["RS_archive_file"] = rs_filepath
```

Comment:
How about just **always** setting the archive filepath _even_ if the setting is enabled. So that if one would want to enable it later they just need to enable the archive option?

Reply:
totally, agreed!

Reply:
yes, since with the node created by AYON, one should expect paths to be managed by it anyway